x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -270,7 +281,7 @@ export class HeadlessChromiumDriverFactory {
   }
 
   getPageExit(browser: any, page: any) {
-    const pageError$ = Rx.fromEvent<Error>(page, 'error').pipe(
+    const pageError$ = this.getPageEventAsObservable(page, 'error').pipe(
```

Comment:
Oops, this looks correct in this PR, but it doesn't match the PR to `main`:

https://github.com/elastic/kibana/pull/177940/files#r1521852726